packages/components/src/internal/components/forms/QueryInfoForm.tsx

```diff
@@ -169,7 +169,9 @@ export class QueryInfoForm extends PureComponent<QueryInfoFormProps, State> {
                     // They are converted back to Dates when saving to the server.
                     const col = this.props.queryInfo?.getColumn(key);
                     if (submitForEdit && col?.jsonType === 'date') {
-                        filteredData = filteredData.set(key, formatDateTime(data[key], null, col.format));
+                        if (col.isDateOnlyColumn)
```

Comment:
Will the `col.jsonType` ever be `time`?

Reply:
`time` needs no special treatment will be handled by the final `else` block